Plugins/BezierSolution/BezierSpline.cs

```diff
@@ -75,6 +75,9 @@ public SplineAutoConstructMode autoConstructMode
 				}
 			}
 		}
+		[HideInInspector] private Vector3[] m_rhs;
```

Comment:
I've used "m_" notation in properties' backing fields only, please rename this to autoConstructedSplineRhs.